cmd/cloud_run.go

```diff
@@ -150,8 +156,13 @@ func (c *cmdCloudRun) flagSet() *pflag.FlagSet {
 		&c.noUsageReport,
 		"no-usage-report",
 		c.noUsageReport,
-		"only when using the local-execution mode, don't send anonymous usage "+
-			"stats (https://grafana.com/docs/k6/latest/set-up/usage-collection/)",
+		"only when using the local-execution mode, don't send anonymous stats to the developers",
```

Comment:
can we keep the link to the docs here for the usage?

Reply:
Indeed, lost in rebase 👍🏻 Addressing it 🙇🏻